src/components/nodes/boundaryEscalationEvent/index.js

```diff
@@ -0,0 +1,34 @@
+import component from './boundaryEscalationEvent.vue';
+import boundaryEventConfig from '../boundaryEvent';
+import merge from 'lodash/merge';
+import cloneDeep from 'lodash/cloneDeep';
+import interruptingToggleConfig from '../boundaryEvent/interruptingToggleInspector';
+
+export default merge(cloneDeep(boundaryEventConfig), {
+  id: 'processmaker-modeler-boundary-escalation-event',
+  component,
+  label: 'Boundary Escalation Event',
+  icon: require('@/assets/toolpanel/boundary-escalation-event.svg'),
+  definition(moddle, $t) {
+    return moddle.create('bpmn:BoundaryEvent', {
+      name: $t('New Boundary Escalation Event'),
+      cancelActivity: true,
+      eventDefinitions: [
+        moddle.create('bpmn:EscalationEventDefinition'),
+      ],
+    });
+  },
+  inspectorConfig: [
+    {
+      items: [
+        {
+          items: [
+            {},
+            {},
+            interruptingToggleConfig,
```

Comment:
@JCook21 not sure how far to extract this. I could go as far as:
```javascript
{
  items: [
          {},
          {},
          {
            component: 'FormCheckbox',
            config: {
              label: 'Interrupting',
              name: 'cancelActivity',
              helper: 'Boundary Event Type',
            },
          },
        ],
},
```

Reply:
Left it as is. Open to suggestions for writing this in a more sharable manner. @tuantruong625 @chipit24